docs/tutorials/gettingstartedexample/gse-gen-common.md

```diff
@@ -16,16 +13,133 @@ This document covers a range of common scenarios using the PXIe-148X Generation
+	
+3. Set the following controls on the Generation Example GSE VI and leave all other values at their defaults.
+    > Note: VI controls and indicators can be reset to default values by clicking on the **Edit** menu and selecting the **Reinitialize Values to Default** option.
+
+    | Tab            | Setting                 | Value                                                                                                                                                               |
+    |----------------|-------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+    | Resource       | RIO Device              | [System Specific]                                                                                                                                                   |
+    | Resource       | Bitfile Path            | [Refer to Bitfile Path in the PXIe-148X Acquisition GSE Help](../../reference/gettingstartedexample/gse-acq-help.md#table-of-pxie-148x-acquisition-bitfiles)        |
+    | Resource       | Display Acquired Images | Disabled                                                                                                                                                            |
+    | Resource       | Log I2C to Disk         | Enabled                                                                                                                                                             |
+	| Serial Channel | Configuration Script    | [Refer to Configuration Script in the PXIe-148X Acquisition GSE Help](../../reference/gettingstartedexample/gse-acq-help.md#table-of-pxie-148x-acquisition-scripts) |
+    
+    | Acquisition    | Continuous Acquisition  | Disabled                                                                                                                                                            |
+    | Board          | Power Over Coax Source  | Internal                                                                                                                                                            |
+4.  Select the **I2C** tab and make the following modifications.
+
+    > The I2C tab only has one control, the I2C **timestamp filter**. This filter contains an array of timestamp IDs. **User24** represents the I2C traffic on serial channel 0 (SI0), **User25** represents the I2C traffic on serial channel 1 (SI1), and so on.
+    - Set the **timestamp filter** array to contain only the **User24** timestamp ID. This will let you see the I2C traffic on SI0. If you run the VI with a configuration script selected, you will see configuration traffic in the I2C Data Output tab after the VI has stopped.
+
+    ![Configuration Settings I2C Tab](../../images/PXIe-148X-Acq-I2CTab-FiniteAcq.png)
+
+5.  Run the Acquisition VI. 
+
+6.  Run the Generation VI.
```

Comment:
Consider whether the same changes should be made as described in my earlier comment (flip the order, add explicit step for clicking the Serializer Setup Complete button)

Reply:
In this case the script is being run from the generation side, so there is no waiting for serializer setup. The generation VI won't wait at all, so it needs to be run second.

Reply:
Good point! I retract. :) Is it worth adding a small explanatory note after those two steps that explain exactly why that happens? Something like this maybe?

Note: Because a script was defined in the **Serializer (Output) Configuration Script** control, the generation begins immediately without pausing while illuminating **Waiting for Serializer Setup** as seen in previous sections of this tutorial.  In this scenario, the Acquisition VI must be started before the Generation VI.

This is optional recommendation. I'm mentioning it because it's tripped me up in the past before 'how things work' finally clicked.